modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/_masturbation_item.dm

```diff
@@ -1,39 +1,37 @@
+#define CUM_VOLUME_MULTIPLIER 5 + 5
+
 /obj/item/hand_item/coom
 	name = "cum"
 	desc = "C-can I watch...?"
 	icon = 'icons/obj/hydroponics/harvest.dmi'
 	icon_state = "eggplant"
 	inhand_icon_state = "nothing"
 
-/obj/item/hand_item/coom/attack(mob/living/target, mob/user, proximity)
-	. = ..()
-	do_masturbate(target, user, proximity)
-
-// Jerk off into bottles
+// Jerk off into bottles and onto people.
 /obj/item/hand_item/coom/afterattack(obj/target, mob/user, proximity)
 	. = ..()
 	do_masturbate(target, user, proximity)
 
+/// Handles masturbation onto a living mob, or an atom.
+/// Attempts to fill the atom's reagent container, if it has one, and it isn't full.
 /obj/item/hand_item/coom/proc/do_masturbate(obj/target, mob/user, proximity)
 	if (CONFIG_GET(flag/disable_erp_preferences))
 		return
 	if(!proximity)
 		return
-	if(user.stat == DEAD)
+	if(user.stat >= DEAD)
 		return
```

Comment:
You could put these three checks on the same line with `||` between them, I don't really see any real reason why they should be separate.

Also add an empty newline after the last return.

Reply:
Because I smart